packages/stores/types.ts

```diff
@@ -305,11 +306,47 @@ type TUiStore = {
     toggleNeedRealAccountForCashierModal: () => void;
     populateHeaderExtensions: (header_items: JSX.Element | null) => void;
     populateSettingsExtensions: (menu_items: Array<TPopulateSettingsExtensionsMenuItem> | null) => void;
+    purchase_states: boolean[];
     setShouldShowCooldownModal: (value: boolean) => void;
 };
 
+type TPortfolioPos = {
```

Comment:
```suggestion
type TPortfolioPos = {
```
Can we give it a better and more readable name?

Reply:
Is `TContractInfoPosition` okay?
@hamza-deriv @farzin-deriv